Source/Core/DiscIO/CISOBlob.cpp

```diff
@@ -44,7 +44,7 @@ CISOFileReader* CISOFileReader::Create(const std::string& filename)
 
 u64 CISOFileReader::GetDataSize() const
 {
-	return GetRawSize();
+	return CISO_MAP_SIZE * m_block_size;
```

Comment:
Should just return the size of a disk: 1459978240

I get the feeling m_block_size will always be the same.